addons/account/models/account_move.py

```diff
@@ -3594,9 +3594,9 @@ def _compute_tax_line_id(self):
     @api.depends('move_id.move_type', 'tax_ids', 'tax_repartition_line_id')
     def _compute_tax_tag_invert(self):
         for record in self:
-            if not record.tax_repartition_line_id and not record.tax_ids:
-                # We only want to set this field to True on lines using taxes
-                record.tax_tag_invert = False
+            if not record.tax_repartition_line_id and not record.tax_ids :
+                # Invoices imported from other softwares might only have kept the tags, not the taxes.
+                record.tax_tag_invert = record.tax_tag_ids and record.moved_id.is_inbound()
```

Comment:
`include_receipts=True` ?

Reply:
It's True by default, isn't it ? :p

Reply:
`record.moved_id` typo, it's traceback-ing